README.md

```diff
@@ -1,3 +1,6 @@
+CloundFront link https://db5z5p02d6kgh.cloudfront.net/
+S3 link https://aws-react-bucket-vsorokin.s3.amazonaws.com/
```

Comment:
It does not show SPA app, only message NoSuchBucket

Reply:
you may recheck it one more tim please, becasue I updated links